.github/workflows/master.yaml

```diff
@@ -0,0 +1,26 @@
+name: master
+on:
+  push:
+    branches:
+      - master
+env:
+  GO111MODULE: on
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2-beta
+      with:
+        go-version: '^1.14'
+    - run: go version
+    - run: sudo apt-get install redis-server
+    - run: make deps check-race cicheck check-plugins
```

Comment:
maybe we can add check-plugins as separate step, and remove the rest together with the condition?

Reply:
I re-arranged it and forced pushed. Can you reply on the `master.yaml` file which steps I should remove? Then I'll mark this comment as resolved.